src/types/web-compat.ts

```diff
@@ -0,0 +1,46 @@
+// eslint-disable
+/**
+ * @module Webcompat Messages Schema
+ * @description
+ * These types are auto-generated from schema files.
```

Comment:
Could these belong in the `build` dir to add clarity they're somewhat temporal based on other files?
I'm guessing that might be an issue due to files might need to be referenced prior to the final build.
My other suggestion would be perhaps to 'generated-types' or similar.

Reply:
+1, would be nice to separate auto-generated files in the folder structure. `build` may be confusing (we probably want to commit these files so that code is traversable in Github), but a separate `generated-types` sgtm

Reply:
right now it's inside `src/types`, because that's where other auto-generated types live. 

I don't imagine this is a blocker - we can talk about moving them all within another PR :)